docs/toc.md

```diff
@@ -84,6 +84,7 @@
 * [Contributing](#)
     * [Getting started](contributing/getting-started.md)
     * [Build instructions](contributing/build-instructions.md)
+    * [Developer tools](contributing/developer-tools.md)
```

Comment:
Shall we soak test these tools internally for a bit first before recommending them to the public?

Reply:
Maybe I keep the page but don't link to TOCs?

Reply:
SGTM

Reply:
Done.